src/Serilog.Extensions.Logging/Extensions/Logging/LevelConvert.cs

```diff
@@ -36,7 +36,8 @@ public static LogEventLevel ToSerilogLevel(LogLevel logLevel)
     {
         return logLevel switch
         {
-            LogLevel.None or LogLevel.Critical => LogEventLevel.Fatal,
+            LogLevel.None => LevelAlias.Off,
```

Comment:
bugfix?

Reply:
I don't think it'll have any effect in the way it's used, here, since events should never carry `None` - just makes the code clearer.